stable/search-prod/templates/operator-role.yaml

```diff
@@ -2,10 +2,10 @@
 # Copyright Contributors to the Open Cluster Management project
 
 apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
+kind: ClusterRole
 metadata:
   creationTimestamp: null
-  name: search-operator
+  name: {{ .Values.org }}:{{ .Release.Name }}:search-operator
```

Comment:
@zhiweiyin318 We have to update this new name clusterrole and clusterrolebinding in search-operator deployment yaml - its still using search-operator